openlibrary/macros/StarRatingsStats.html

```diff
@@ -6,7 +6,11 @@
 <ul class="readers-stats" itemprop="aggregateRating" itemscope itemtype="https://schema.org/AggregateRating">
   <li class="avg-ratings">
     $if rating_stats.get('avg_rating'):
-      $:('<span class="readers-stats__star">★</span>' * int(rating_stats.get('avg_rating'))) <span itemprop="ratingValue">$(rating_stats['avg_rating'])</span>
+      $ stats_decimal = (float(rating_stats.get('avg_rating')))-(int(rating_stats.get('avg_rating')))
```

Comment:
Rather than repeating rating_stats.get('avg_rating')
I'd suggest reading it and caching it once.

e.g. 
```
$ avg = rating_stats.get('avg_rating')

```

Reply:
I have made the changes as requested